test/Guzzle/SubjectRewritingMiddlewareTest.php

```diff
@@ -387,6 +387,68 @@ public function responseProvider() : Traversable
                     ],
                 ],
             ],
+            'application/vnd.elife.search+json; version=1; without new subject' => [
+                [
+                    'total' => 2,
+                    'items' => [
+                        $this->createArticlePoA(false),
+                        $this->createArticleVoR(false),
+                    ],
+                    'subjects' => [
+                        $this->createSubject('old-subject', 'Old Subject') + ['results' => 2],
+                        $this->createSubject('other-subject', 'Other Subject') + ['results' => 8],
+                    ],
+                    'types' => [
```

Comment:
the value of `types` could be extracted somewhere since it's not changing in the rewrite? It makes for a faster comparison of before and after